.flowconfig

```diff
@@ -19,6 +19,8 @@ module.name_mapper.extension='scss' -> '<PROJECT_ROOT>/packages/webviz-core/src/
 module.name_mapper.extension='md' -> '<PROJECT_ROOT>/packages/webviz-core/src/types/modules/Markdown.js.flow'
 module.name_mapper.extension='ne' -> '<PROJECT_ROOT>/packages/webviz-core/src/types/modules/Nearley.js.flow'
 module.name_mapper.extension='worker.js' -> '<PROJECT_ROOT>/packages/webviz-core/src/types/modules/WorkerLoader.js.flow'
+module.name_mapper='stories/\(.*\)' -> '<PROJECT_ROOT>/stories/\1'
+module.name_mapper='common/\(.*\)' -> '<PROJECT_ROOT>/common/\1'
```

Comment:
Should we instead add `module.system.node.resolve_dirname=.` for consistency with https://github.com/cruise-automation/webviz/pull/212/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR69?